client/src/components/CategoryIndex.js

```diff
@@ -27,10 +28,13 @@ const CategoryIndex = props => {
 
   const categoryTiles = categories.map(categoryObject => {
     return (
-      <CategoryTile 
-        key={categoryObject.id}
-        category={categoryObject.name}
-      />
+      <div key={categoryObject.id}>
```

Comment:
we should also always avoid passing a key to a JSX element. This is actually code smell, for moving the div into the CategoryTile itself, and passing the key instead as a prop to category

in this case, after making the change suggested below, you can delete the div on line 31 altogether